### PR TITLE
[#1070] Disable File Uploads

### LINKED ISSUE #1070: Disable Uploads
Switch to Prism, October 23, 2022.

We should add a banner to notify users of some info.

### PR DESCRIPTION
**Pull request by Meowcenary**
Disable file uploads by setting max allowed files to zero, max file size to zero bytes, disabling the buttons that are used for file upload, and removing all links to the file upload page. closes #1070

### CODE REVIEW

File: app/views/generic_files/_browse_everything.html.erb
Comment (Meowcenary) [inline app/views/generic_files/_browse_everything.html.erb:1]: This and the other empty view are here to override the views that are provided by sufia

File: app/views/dashboard/_index_partials/_heading_actions.html.erb
Comment (Meowcenary) [inline app/views/dashboard/_index_partials/_heading_actions.html.erb:1]: This is to override the sufia provided view

File: app/assets/javascripts/files.js
Comment (Meowcenary) [inline app/assets/javascripts/files.js:1]: Disabling the file upload

**Review by fenekku — APPROVED** [review]
Works for me. People should still be able to edit (which is what we want, and in an ideal world, want only for librarians but that's ok), right?

Obviously this should not be merged/deployed until the opportune moment.

**Comment by Meowcenary** [comment]
> Works for me. People should still be able to edit (which is what we want, and in an ideal world, want only for librarians but that's ok), right?
> 
> Obviously this should not be merged/deployed until the opportune moment.

From what I saw the editing would still work fine. Yes, we're just going to have this ready for when the time comes.

**Review by carsonicator — APPROVED** [review]
Thanks @Meowcenary . Yes, ideally, we would like to allow edit access only to certain users after these "read-only" changes are made.

**Comment by mattbaumann1** [comment]
@Meowcenary went to look at another pull request and noticed this pull request hanging out. Is a review on this still needed?

**Comment by fenekku** [comment]
This PR is just waiting on the go from the decision team to make the transition to Prism before being merged. When it's merged and deployed, DigitalHub new entries will be disabled.

**Review by mattbaumann1 — APPROVED** [review]

**Comment by Meowcenary** [comment]
Going to rebase this today since it has been sitting out here for awhile. I don't think there will be conflicts, but better to just stay up to date.

**Comment by Meowcenary** [comment]
I just rebased this so it should be good to go whenever we want to disable uploading. I'll test it out after I finish changes for #1117

**Pull request closed without merging**
